includes/HttpFunctions.php

```diff
@@ -77,6 +77,9 @@ public static function request( $method, $url, $options = array() ) {
 		// log all the requests we make
 		$caller =  wfGetCallerClassMethod( [ __CLASS__, 'Hooks', 'ApiService', 'Solarium_Client', 'Solarium_Client_Adapter_Curl', 'ExternalHttp' ] );
 		$isOk = $status->isOK();
+
+		wfRunHooks( 'HttpRequestAfter', [ $method, $url, $caller, $requestTime, $req->getResponseHeaders() ] ); # Wikia change
```

Comment:
`AfterHttpRequest` ?
(AFAIR the `After` and `Before` are exception and go at the beginning of the hook name according to the convention)